test/docfx.Test/lib/JavascriptEngineTest.cs

```diff
@@ -39,5 +39,15 @@ public void RunJavascriptError(string input)
                 Assert.Throws<JavaScriptEngineException>(() => engine.Run("index.js", "main", inputJson));
             }
         }
+
+        [Fact]
+        public void SupportModuleExports()
```

Comment:
Supports NodeJS `module.exports` standard so we can `require('./lunr.js')` properly.